backend/src/main/kotlin/org/loculus/backend/controller/SubmissionController.kt

```diff
@@ -252,16 +256,30 @@ class SubmissionController(
     fun getReleasedData(
         @PathVariable @Valid organism: Organism,
         @RequestParam compression: CompressionFormat?,
+        @RequestHeader(value = HttpHeaders.IF_MODIFIED_SINCE, required = false) ifModifiedSince: Long?,
```

Comment:
If we pass a long we are kind of violating HTTP specs which have a specific (annoying) format for this particular header. We might want to choose our own header field if we are not spec compliant to not confuse people.

We probably want extra millisecond precision here just to make sure we don't miss an update.

To make a potential cache race condition less impactful, we could always request all data after time X, say 1hr, then we are at most that much out of date, but this is best handled in client I think

Reply:
Ah true - I see that the expected input is: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/If-Modified-Since#syntax, so we could check if there is a way to convert a string to a sql.Timestamp object and then compare.

Reply:
Or yeah we just make our own header field with unix timestamp - that is easiest